third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -1664,6 +1695,15 @@ impl Stack {
         Ok(args)
     }
 
+    fn last_n_tys_mut(&mut self, n: usize) -> PartialVMResult<&mut [Type]> {
+        if self.value.len() < n {
```

Comment:
You should check `self.types.len()` here, if for whatever reason stacks becomes unbalanced, this may panic otherwise